### DISCUSSION TITLE
Adding or linking tables making multiple relationship database

### DISCUSSION THREAD

**Discussion by rcbledsoe** [sqlitebrowser/sqlitebrowser#3813] [Ideas]
Can this sqlitebrowser links many tables if needed?

**Comment by justinclift** [comment]
When you say "link many tables", are you meaning "Referential Integrity"?

If that's what you mean, then yes it can.  You have to enable it though, by using the `foreign_keys` PRAGMA like mentioned here: https://www.sqlite.org/foreignkeys.html#fk_enable
